bucket/git-filter-repo.json

```
@@ -7,12 +7,18 @@
         "Git": "git",
         "Python 3": "python"
     },
-    "url": "https://github.com/newren/git-filter-repo/archive/v2.47.0.zip#/dl.7z",
+    "url": "https://github.com/newren/git-filter-repo/archive/v2.47.0.zip#/dl.bin",
     "hash": "8f3d099bf91ceac6cae60d79aac701e9bc08fd0eed67c601cc5dd401d5788871",
-    "pre_install": "Move-Item \"$dir\\git-filter-repo-*\\*\" \"$dir\"; Remove-Item \"$dir\\git-filter-repo-*\"",
+    "installer": {
+        "script": [
+            "Expand-ZipArchive -Path \"$dir\\dl.bin\" -DestinationPath \"$dir\" -Removal",
+            "Move-Item \"$dir\\git-filter-repo-*\\*\" \"$dir\"; Remove-Item \"$dir\\git-filter-repo-*\"",
+            "Copy-Item \"$dir\\git-filter-repo\" \"$dir\\contrib\\filter-repo-demos\" | Out-Null"
```

```suggestion
            "Expand-ZipArchive -Path \"$dir\\dl.zi_\" -DestinationPath \"$dir\" -Removal",
            "Move-Item \"$dir\\git-filter-repo-*\\*\" \"$dir\"; Remove-Item \"$dir\\git-filter-repo-*\""
```

The last command is not needed now imo.

---

I don't think that's required either.